### ISSUE TITLE
Multichannel Flac files not loading

### ISSUE THREAD

**Issue by Roadrider-Projects** [jherby2k/AudioWorks#33]
Follolowing error on all my Multichannel Flac files. (4, 5, 6 and 8 channels)

> $af = Get-AudioFile G:\tools\test2.flac
**Get-AudioFile : Unable to decode 'G:\tools\test2.flac' with any loaded extension.
At line:1 char:8
+  $af = Get-AudioFile G:\tools\test2.flac
+        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (G:\tools\test2.flac:String) [Get-AudioFile], AudioUnsupportedException
    + FullyQualifiedErrorId : AudioUnsupportedException,AudioWorks.Commands.GetAudioFileCommand**

I don't know how to checkt the version. Instaled AudioWorks using "Install-Module -Name AudioWorks.Commands" today (12 - Feb - 2021) (expect 1.1.0)

This Flac File works with DBPoweramp, Flac.exe,  MetaFlac.exe and TagLib.dll

AudioWorks works fine with my 2 Channel .Flac files.

**Comment by jherby2k** [comment]
Yeah, multi-channel isn't supported yet. I should make the AudioUnsupportedException more clear.

**Comment by BenMcLean** [comment]
> Yeah, multi-channel isn't supported yet. I should make the AudioUnsupportedException more clear.

Oh no, that means I'm using the wrong library entirely.

I'd like to know when this library gets multi-channel support and has some decent example code of how to do basic things. Like please illustrate the life cycles of your classes and how they relate to one another.
